### PR TITLE
fix(PaginatedMessage): I couldn't take it, sorry

### PR DESCRIPTION
**Pull request by kaname-png**

### CODE REVIEW

File: packages/discord.js-utilities/src/lib/PaginatedMessages/PaginatedMessage.ts [hunk 1]
```diff
@@ -739,8 +739,8 @@ export class PaginatedMessage {
 		for (const [idx, embed] of Object.entries(data.embeds)) {
 			if (embed) {
 				embed.footer ??= { text: this.template.embeds?.[Number(idx)]?.footer?.text ?? this.template.embeds?.[0]?.footer?.text ?? '' };
-				embed.footer.text = `${this.pageIndexPrefix ? `${this.pageIndexPrefix} ` : ''}${index + 1} / ${this.pages.length}${
-					embed.footer.text ? ` • ${embed.footer.text}` : ''
+				embed.footer.text = `${this.pageIndexPrefix ? `${this.pageIndexPrefix} ` : ''}${index + 1} / ${this.pages.length} | ${
```
Comment (kyranet) [inline packages/discord.js-utilities/src/lib/PaginatedMessages/PaginatedMessage.ts:742]: The idea is that the `|` is only appended when there's a footer text.
```suggestion
				embed.footer.text = `${this.pageIndexPrefix ? `${this.pageIndexPrefix} ` : ''}${index + 1} / ${this.pages.length}${
```

**Review by kyranet — CHANGES_REQUESTED** [review]

File: packages/discord.js-utilities/src/lib/PaginatedMessages/PaginatedMessage.ts [hunk 2]
```diff
@@ -739,8 +739,8 @@ export class PaginatedMessage {
...
+					embed.footer.text ? ` | ${embed.footer.text}` : ''
```
Comment (jurienhamaker) [inline packages/discord.js-utilities/src/lib/PaginatedMessages/PaginatedMessage.ts:743]: Might as well make it configureable kekw

**Review by Favna — CHANGES_REQUESTED** [review]
I don't get it at all. Plus Kyra's comment needs to be resolved.

**Comment by Favna** [comment]
Closing because this what this PR was meant to fix, was actually already fixed in a4e5a567a4d75dbba57df6774664ebaff55c24f4

**Pull request closed without merging**